src/ingest_validation_tools/directory-schemas/imc3d.yaml

```diff
@@ -1,13 +1,13 @@
 # mcd/:
 -
-  pattern: 'mcd/[^/]+_HuBMAP_[^/]+organ[^/]+_slide[^/]+\.zip'
-  description: 'csv containing labels for sections as well as whether or not they were included in the 3D model'
+  pattern: 'mcd/[^/]+_HuBMAP_[^/]+_slide[^/]+\.zip'
+  description: 'CSV containing labels for sections as well as whether or not they were included in the 3D model'
   required: True
   is_qa_qc: False
 
 -
   pattern: 'mcd/section_report\.csv'
-  description: 'csv containing labels for sections as well as whether or not they were included in the 3D model'
+  description: 'CSV containing labels for sections as well as whether or not they were included in the 3D model'
```

Comment:
What are the columns?

Reply:
@mccalluc updated the column info today.